### DISCUSSION TITLE
4.7.7

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#917] [Announcements]
#### ADDED
- 推荐页视频右键快速设置过滤up主 #912
- 历史记录页加入观看到的时间点和看完与否，加入显示准确时间的tooltip
    - 感谢 @GD-Slime
#### CHANGED
- 调整评论区样式与个人空间页面样式，修复个人空间页动态评论背景透明问题 #905 #898
    - 感谢 @ProJend
- 搜索页样式调整 #909 #898
    - 感谢 @ProJend
- 主题颜色样式调整 #913 #898
    - 感谢 @ProJend
#### FIXED
- 搜索历史记录的api错误 #896
- 处理标签页TabViewList最大宽度，修复状态栏卡片右上角与功能键重叠 #900
- 修复单窗口模式全屏播放窗口标题栏不消失 #897
- 修复视频详情页右侧信息透明问题 #910

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.7'>4.7.7</a>.</em>

**Comment by KennethWuWu** [comment]
大佬，新问题x86版本全屏播放后会音画不同步，半屏播放的时候不会有这个问题。x64版本的也没有这个问题。

**Reply by ywmoyue** [reply]
@KennethWuWu 设置里面播放器选择的哪一个，选择另一个试试？

![image](https://github.com/user-attachments/assets/afdb1a38-fc4d-4139-8403-8d02ee573adc)

**Reply by KennethWuWu** [reply]
切换了也不行，反而问题变得更多了，加载慢还卡顿。暂时退回7.4版本了，7.4一切正常。

**Reply by ywmoyue** [reply]
播放器没改动影响播放的内容，下个临时版本我把播放器相关的改动去掉试试

**Reply by KennethWuWu** [reply]
好的，大佬辛苦了。

**Reply by ywmoyue** [reply]
@KennethWuWu 麻烦试一下https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.7.801

**Reply by KennethWuWu** [reply]
> @KennethWuWu 麻烦试一下https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.7.801

这个版本没有问题了大佬。😊

**Comment by KennethWuWu** [comment]
大佬，4.7.7.806版本还是老问题，x86版本的详情页一团黑。

**Reply by ywmoyue** [reply]
我看下，我记得好像没改

**Reply by ywmoyue** [reply]
找到哪里的改动可能有影响了，下个临时版本打个日志看下

**Reply by KennethWuWu** [reply]
好的

**Reply by ywmoyue** [reply]
@KennethWuWu 请问主题是设置的跟随系统还是浅色\深色

**Reply by KennethWuWu** [reply]
> @KennethWuWu 请问主题是设置的跟随系统还是浅色\深色

跟随系统，之前也试过调成浅色模式，详情页都是一团黑。

**Reply by ywmoyue** [reply]
@KennethWuWu https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.7.807 试试设置浅色\深色模式，跟随系统的我复现了，浅色\深色模式现在在我这边没问题

**Reply by KennethWuWu** [reply]
> @KennethWuWu https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.7.807 试试设置浅色\深色模式，跟随系统的我复现了，浅色\深色模式现在在我这边没问题

我又试了一下，确实是跟随系统下会变成一团黑，调成浅色/深色模式就正常了。😄
